folium/folium.py

```diff
@@ -664,6 +666,31 @@ def fit_bounds(self, bounds, padding_top_left=None,
         self.template_vars.update({'fit_bounds': fit_bounds_str})
 
 
+    def _auto_bounds(self):
+        if 'fit_bounds' in self.template_vars:
```

Comment:
I am wondering if we should add an option here to fit just an specific feature, like markers, instead of everything.

Here is what got me thinking.  I have an application where I add several markers, lines, etc.  But I want the map to autofit only on the markers (ocean gliders that are moving in time).  That way the user will always get a map centred on them.

On the other hand I like to keep things simple and your current implementation is exactly what most users expect from an autofit.

I am on the fence here. @birdage what do you think?